docs/polymorphism.rst

```diff
@@ -0,0 +1,36 @@
+Polymorphism
+============
+
+PynamoDB supports polymorphism through the use of discriminators.
+
+A discriminator is a value that is written to DynamoDB that identifies the python class being stored.
+(Note: currently discriminators are only supported on MapAttribute subclasses; support for model subclasses coming soon.)
+
+Discriminator Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The discriminator value is stored using a special attribute, the DiscriminatorAttribute.
+Only a single DiscriminatorAttribute can be defined on a class.
+
+The discriminator value can be assigned to a class as part of the definition:
+
+.. code-block:: python
+
+    class ParentClass(MapAttribute):
+        _cls = DiscriminatorAttribute()
+
+    class ChildClass(ParentClass, discriminator='child'):
```

Comment:
Should we add an explicit warning about the discriminator value being stored in DDB, so changing the value would require a migration (or additional care with model registration)? I know it seems obvious, but someone is gonna get this wrong